docs/src/config.md

```diff
@@ -235,10 +235,16 @@ required. A `reducer` can be specified to apply to the model output, see for mor
 information the following section [Output CSV section](@ref). When a `map` is provided to
 extract data for certain locations (e.g. `gauges`) or areas (e.g. `subcatchment`), the
 NetCDF location names are extracted from these maps. For a specific location (grid cell) a
-`location` is required. In the section [Output CSV section](@ref), similar functionality is
-available for CSV. For integration with Delft-FEWS, see also [Run from Delft-FEWS](@ref),
-it is recommended to write scalar data to NetCDF format since the General Adapter of
-Delft-FEWS can ingest this data format directly.
+`location` is required. For layered model parameters and variables that have an extra
+dimension `layer` and are part of the vertical `sbm` concept it is possible to specify an
+internal layer index (see also example below). If multiple layers are desired, this can be
+specified in separate `[[netcdf.variable]]` entries. Note that the specification of the
+layer is not optional when Wflow is integrated with Delft-FEWS, for NetCDF scalar data an
+extra dimension is not allowed by the `importNetcdfActivity` of the Delft-FEWS General
```

Comment:
Oh I thought with `axis = "Z"` and float coordinates it would work? I see in the code that for netCDF scalar output retaining the layer dimension is supported.

Reply:
For NetCDF scalar data this is not supported (tested this and also asked Onno about this), so the `axis = "Z"` only works for 3D data. So when linking to FEWS, a user needs to specify a `layer`.

Reply:
Ah yes I see, that was grid data only, thanks